scripts/eve-embedded.sh

```diff
@@ -41,10 +41,8 @@ bootstrap() {
 
 # if this is the first run on this /persist -- generate everything
 if [ ! -d $DB ]; then
-   adam generate --db-url $DB --server --hosts 127.0.0.1,localhost --cn localhost.localdomain
-   cp $DB/server.pem /config/root-certificate.pem
-   echo $SERVER > /config/server
```

Comment:
Isn't removing this going to break it? The default is not `localhost:6000`

Reply:
It is very much localhost:6000 in the embedded case

Reply:
Where is that set?